### DISCUSSION TITLE
Havit Grid -Page Size Dropdown? Total Record Count?

### DISCUSSION THREAD

**Discussion by chobo2** [havit/Havit.Blazor#770] [Q&A]
Hi

I am wondering if I am not seeing it or if it just not a feature and I would need to roll out my own.

I don't see an option to add a Page Size dropdown (like 5, 10 ,50, 100, All) or something showing that you're seeing 10 records of 5000.

Is the expectation to implement myself?

**Comment by hakenr** [comment]
Such features are currently not built into HxGrid, you can add them easily beside the HxGrid.
Item in our backlog (not decided yet): https://github.com/havit/Havit.Blazor/issues/96

**Reply by chobo2** [reply]
Great, just wanted to double-check before I go and implement it myself if business decides it is needed.
